tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -45,15 +45,21 @@ def test_dcv_configuration(
     # dcv connect show url
     env = operating_system.environ.copy()
     env["AWS_DEFAULT_REGION"] = region
-    # Disable StrictHostKeyChecking by adding option to end of file
-    operating_system.system(r"echo '    StrictHostKeyChecking no' >> /etc/ssh/ssh_config")
-    result = run_command(["pcluster", "dcv", "connect", cluster.name, "--show-url"], env=env)
+
+    # add ssh key to global known hosts file to avoid ssh keychecking prompt
+    host_keys_file = operating_system.path.expanduser("~/.ssh/known_hosts")
+    add_keys_to_known_hosts(cluster.master_ip, host_keys_file)
+
+    try:
+        result = run_command(["pcluster", "dcv", "connect", cluster.name, "--show-url"], env=env)
+    finally:
+        # remove ssh key from global known hosts file
+        remove_keys_from_known_hosts(cluster.master_ip, host_keys_file, env=env)
```

Comment:
is there any risk of multiple processes trying to modify the hosts file at the same time?

Reply:
This should not be a problem, because to add keys, we are only appending to file and not overwriting. To remove, `ssh-keygen` scans the file and only removes the matching entries. So multiple add/remove at the same time should not interfere with each other.

Passing Jenkins test had 2 tests in parallel and it seems to be fine.

Reply:
Okay. I do think this could cause issues if we get unlucky, but I agree that the chances of it happening are very small considering only two of these tests will ever be running at one time.